oxidation/parsec_cli/parsec_sequestre_dump/src/main.rs

```diff
@@ -0,0 +1,176 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+#[macro_use]
+extern crate diesel;
+
+mod model;
+mod schema;
+
+use clap::Parser;
+use diesel::{Connection, SqliteConnection};
+use std::{error::Error, path::PathBuf};
+
+use crate::model::Workspace;
+use parsec_api_crypto::SecretKey;
+
+/// Simple program that dumps files and folders in a file
+#[derive(Parser, Debug)]
+#[clap(version, about)]
+struct Args {
+    /// Sqlite input file (e.g `my/workspace_export.sqlite`)
+    #[clap(short, long)]
+    input: PathBuf,
+
+    /// Secret key (e.g `my/keyfile.key`)
+    #[clap(short, long)]
+    key: String,
+
+    /// Output file (e.g `./output`)
+    #[clap(short, long)]
+    output: PathBuf,
+}
+
+fn main() -> Result<(), Box<dyn Error>> {
+    let args = Args::parse();
+
+    let input = args.input.to_str().unwrap();
+    let key_path = args.key.as_str();
+    let output = args.output.to_str().unwrap();
+
+    let mut conn = SqliteConnection::establish(input)?;
+
+    let key_file = std::fs::read(key_path)?;
+    let key = SecretKey::try_from(&key_file[..])?;
+    let workspace = Workspace::query_all(&mut conn, &key)?;
+
+    workspace.dump(output)
+}
+
+#[cfg(test)]
+mod tests {
+    use diesel::{Connection, SqliteConnection};
+    use rstest::rstest;
+    use tests_fixtures::{alice, Device};
+
+    use crate::model::{Data, Workspace};
+
+    #[rstest]
+    fn test_parsec_sequestre_dump(alice: &Device) {
+        use diesel::connection::SimpleConnection;
+
+        let input = "/tmp/parsec_sequestre_dump.sqlite";
```

Comment:
we need a `tmp_path` fixture that provide a temporary folder instead of always writing in the same place (this is bad because it can lead do concurrent issues, especially with multiple tests using that all initialize with the same location due to copy/paste coding)

Reply:
and the `tmp_path` will be great to be put in the `tests_fixtures` crate given it most likely is going to be needed in other plenty of other tests ;-)

Reply:
Done in #2218